examples/kubernetes-istio/cilium-kube-inject.awk

```diff
@@ -16,9 +16,9 @@
 /initContainers:/ {
 	indent = $0 ; gsub(/[^ ].*/, "", indent)
 	print indent "- name: sleep"
-	print indent "  image: busybox"
+	print indent "  image: busybox:1.28.4"
 	print indent "  imagePullPolicy: IfNotPresent"
-	print indent "  command: ['sh', '-c', 'sleep 10']"
+	print indent "  command: ['sh', '-c', 'until nslookup kube-dns.kube-system.svc.cluster.local; do sleep 1; done']"
```

Comment:
I'd include a timeout and error out once it's reach to avoid this looping infinitely.

Reply:
This is already better than what we have. I've filed https://github.com/cilium/cilium/issues/6230 to track the timeout addition.